los-deferal-02/Hello-Books
# #166238095 Create Mail Module for Sending Email Messages

#### What does this PR do?
Creates an email module for sending email message to user at any point in the application

#### Description of Task to be completed?
- Use [SendGrid API](https://sendgrid.com) as the email delivery service provider
- Creates `EmailModule` class that contains method for sending the email

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/los-deferal-02/Hello-Books.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd Hello-Books/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/12/head:ft-create-email-module-166238095`
- Now that you have a copy of the branch, switch to it using  `git checkout ft-create-email-module-166238095`
- Your directory will now be an exact copy of the PR
- Run `npm install` to install all project's dependencies onto your local machine

#### Any background context you want to provide?
This is a sub-feature is to be used when implementing features that requires sending email notification to users

#### What are the relevant pivotal tracker stories?
[#166238095](https://www.pivotaltracker.com/story/show/166238095)
